src/rosenv/environment/distro.py

```diff
@@ -109,7 +110,7 @@ class DistroConfig(NamedTuple):
     ),
     builder_tool_variable_name="AMENT_CURRENT_PREFIX",
     meta_package_prevent_overwrite=(
-        "_local_setup_util.py",
```

Comment:
That is weird. I spun up a current `iron` container. There it's still `_local_setup_util.py`.

Was this a miss-commit or do we actually have breaking stuff in different ROS2 versions?